plugins/resource-versions.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: resource-versions
+spec:
+  version: v0.1.1
+  platforms:
+  - bin: kubectl-resource-versions
+    uri: https://github.com/chengshiwen/kubectl-resource-versions/releases/download/v0.1.1/kubectl-resource-versions-darwin-amd64.tar.gz
+    sha256: d236209638d9ad6a001b66ed9c69df483773bc3bfd11aff2dac966774b1ef524
+    files:
+      - from: "kubectl-resource-versions-darwin-amd64/*"
+        to: .
+    selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+  - bin: kubectl-resource-versions
+    uri: https://github.com/chengshiwen/kubectl-resource-versions/releases/download/v0.1.1/kubectl-resource-versions-linux-amd64.tar.gz
+    sha256: 12f7072196d274cb6e08f58cce759b266f299bb9c7e6b04eb127fa9c918763d9
+    files:
+      - from: "kubectl-resource-versions-linux-amd64/*"
+        to: .
+    selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+  - bin: kubectl-resource-versions.exe
+    uri: https://github.com/chengshiwen/kubectl-resource-versions/releases/download/v0.1.1/kubectl-resource-versions-windows-amd64.tar.gz
+    sha256: f44f8c371a779ef988f83eb9c6a971bf02b10391439900ac992601e3ba9675b3
+    files:
+      - from: "kubectl-resource-versions-windows-amd64/*"
+        to: .
+    selector:
+      matchLabels:
+        os: windows
+        arch: amd64
+  shortDescription: Print available versions of each API
+  homepage: https://github.com/chengshiwen/kubectl-resource-versions
+  description: This plugin prints the API resources available on the API server along with groups/versions that offer that API
```

Comment:
Please wrap around at ~80 chars, so that it prints nicely on wide and narrow terminals.

Reply:
Done, the modified description is `Print the API resources along with the supported API versions on the server`